Description: Honour cppflags too
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Index: ndpi/example/Makefile.in
===================================================================
--- ndpi.orig/example/Makefile.in
+++ ndpi/example/Makefile.in
@@ -19,10 +19,10 @@ libndpiReader.a: $(COMMON_SOURCES:%.c=%.
 
 ndpiReader: libndpiReader.a $(LIBNDPI)
 ndpiReader: $(EXECUTABLE_SOURCES:%.c=%.o)
-	$(CC) $(CFLAGS) $(EXECUTABLE_SOURCES:%.c=%.o) libndpiReader.a -o $@ $(LDFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(EXECUTABLE_SOURCES:%.c=%.o) libndpiReader.a -o $@ $(LDFLAGS)
 
 %.o: %.c $(HEADERS) Makefile
-	$(CC) $(CFLAGS) -c $< -o $@
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $@
 
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin/
Index: ndpi/src/lib/Makefile.in
===================================================================
--- ndpi.orig/src/lib/Makefile.in
+++ ndpi/src/lib/Makefile.in
@@ -52,7 +52,7 @@ $(NDPI_LIB_SHARED): $(OBJECTS)
 	ln -fs $(NDPI_LIB_SHARED) $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR).$(NDPI_VERSION_MINOR)
 
 %.o: %.c $(HEADERS) Makefile
-	$(CC) $(CFLAGS) -c $< -o $@ $(LDFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $@ $(LDFLAGS)
 
 clean:
 	/bin/rm -f $(NDPI_LIB_STATIC) $(OBJECTS) *.o *.so *.lo $(NDPI_LIB_SHARED)
