Description: Honour cppflags too
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Index: ndpi/example/Makefile.in
===================================================================
--- ndpi-3.4.orig/example/Makefile.in
+++ ndpi-3.4/example/Makefile.in
@@ -25,13 +25,13 @@ libndpiReader.a: $(COMMON_SOURCES:%.c=%.
 	ar rsv libndpiReader.a $(COMMON_SOURCES:%.c=%.o)
 
 ndpiReader: libndpiReader.a $(LIBNDPI) ndpiReader.o
-	$(CC) $(CFLAGS) ndpiReader.o libndpiReader.a -o $@ $(LDFLAGS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) ndpiReader.o libndpiReader.a -o $@ $(LDFLAGS)
 
 ndpiSimpleIntegration: ndpiSimpleIntegration.o
-	$(CC) $(CFLAGS) $< -o $@ $(LDFLAGS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) $< -o $@ $(LDFLAGS)
 
 %.o: %.c $(HEADERS) Makefile
-	$(CC) $(CFLAGS) -c $< -o $@
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c $< -o $@
 
 install: ndpiReader
 	mkdir -p $(DESTDIR)$(PREFIX)/bin/
--- ndpi-3.4.orig/src/lib/Makefile.in
+++ ndpi-3.4/src/lib/Makefile.in
@@ -58,7 +58,7 @@ $(NDPI_LIB_SHARED): $(OBJECTS)
 	ln -fs $(NDPI_LIB_SHARED) $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR).$(NDPI_VERSION_MINOR)
 
 %.o: %.c $(HEADERS) Makefile
-	$(CC) $(CFLAGS) -c $< -o $@
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c $< -o $@
 
 clean:
 	/bin/rm -f $(NDPI_LIB_STATIC) $(OBJECTS) *.o *.so *.lo libndpi.so*
