From 554f2459a59fbd066710c38a9e9afe33562d2d7b Mon Sep 17 00:00:00 2001
From: Subhajit Chatterjee <subhajit.chatterjee@cdot.in>
Date: Fri, 6 Nov 2020 07:08:56 +0000
Subject: [PATCH] ndpireader-dynamic-link.patch updated

---
 example/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example/Makefile.in b/example/Makefile.in
index ec44acb..e1bff38 100644
--- a/example/Makefile.in
+++ b/example/Makefile.in
@@ -5,7 +5,7 @@ BUILD_MINGW=@BUILD_MINGW@
 SRCHOME=../src
 CFLAGS=-g -fPIC -DPIC -I$(SRCHOME)/include -DHAVE_JSON_C @PCAP_INC@ @CFLAGS@
 LIBNDPI=$(SRCHOME)/lib/libndpi.a
-LDFLAGS=$(LIBNDPI) @PCAP_LIB@ @LIBS@ @ADDITIONAL_LIBS@ -lpthread -lm @LDFLAGS@
+LDFLAGS=-L../src/lib -lndpi @PCAP_LIB@ @LIBS@ @ADDITIONAL_LIBS@ -lpthread -lm @LDFLAGS@
 HEADERS=intrusion_detection.h reader_util.h $(SRCHOME)/include/ndpi_api.h \
         $(SRCHOME)/include/ndpi_typedefs.h $(SRCHOME)/include/ndpi_protocol_ids.h
 OBJS=ndpiReader.o reader_util.o intrusion_detection.o
-- 
2.20.1

