Description: Include the SVN revision in the library version.
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ndpi/configure.ac
===================================================================
--- ndpi.orig/configure.ac
+++ ndpi/configure.ac
@@ -1,4 +1,4 @@
-AC_INIT([nDPI], [1.4.0])
+AC_INIT([nDPI], [1.4.0+svn8032])
 AC_CONFIG_SRCDIR([src/include/ndpi_protocols.h])
 
 AM_INIT_AUTOMAKE([-Wall])
