---
 src/include/ndpi_define.h.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/include/ndpi_define.h.in
+++ b/src/include/ndpi_define.h.in
@@ -256,7 +256,7 @@
 #define NDPI_COMPARE_IPV6_ADDRESS_STRUCTS(x,y)  \
   ((x.u6_addr.u6_addr64[0] < y.u6_addr.u6_addr64[0]) || ((x.u6_addr.u6_addr64[0] == y.u6_addr.u6_addr64[0]) && (x.u6_addr.u6_addr64[1] < y.u6_addr.u6_addr64[1])))
 
-#define NDPI_NUM_BITS              512
+#define NDPI_NUM_BITS              1024
 
 #define NDPI_BITS /* 32 */ (sizeof(ndpi_ndpi_mask) * 8 /* number of bits in a byte */)        /* bits per mask */
 #define howmanybits(x, y)   (((x)+((y)-1))/(y))
